spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -11,6 +11,31 @@
       and_return(true)
   end
 
+  it 'accepts updates and redirects to the right place', allow_browser_log: true do
+    user = user_with_2fa
+
+    sign_in_and_2fa_user(user)
+    begin_in_person_proofing(user)
+    complete_location_step(user)
+    complete_prepare_step(user)
+    complete_state_id_step(user)
+    complete_address_step(user)
+    complete_ssn_step(user)
+
+    expect(page).to have_current_path(idv_in_person_verify_info_path)
+
+    # ASIDE: Hitting 'Back' (test below) redirects to verify instead of verify_info
+    # Do we want to fix that too?
```

Comment:
Discussion point: This isn't covered in the acceptance criteria, and it may be more of a pain for less benefit to conditionalize the back buttons? But I wanted to flag that I noticed this behavior.